05_surface_balance/priestley_taylor.xml

```diff
@@ -336,20 +335,6 @@
           </ParameterList>
         </ParameterList>
       </ParameterList>
-      <ParameterList name="surface-relative_humidity" type="ParameterList">
-        <Parameter name="evaluator type" type="string" value="independent variable" />
-        <ParameterList name="function" type="ParameterList">
-          <ParameterList name="domain rain" type="ParameterList">
-            <Parameter name="region" type="string" value="surface domain" />
-            <Parameter name="component" type="string" value="cell" />
-            <ParameterList name="function" type="ParameterList">
-              <ParameterList name="function-constant" type="ParameterList">
-                <Parameter name="value" type="double" value="0.6" />
-              </ParameterList>
-            </ParameterList>
-          </ParameterList>
-        </ParameterList>
-      </ParameterList>
```

Comment:
These are things going here. First, priestley_taylor uses saturated vapor pressure to calculate the vapor_pressure-temperature slope. Because saturated vapor pressure was changed, the answer would be changed. Second, RH was actually not used for priestley_taylor though the evaluator asks for "relative humidity". I removed this from input.